### ISSUE TITLE
scene detection with node-fluent-ffmpeg

### ISSUE THREAD

**Issue by samuelcalegari** [fluent-ffmpeg/node-fluent-ffmpeg#706]
hello
I try to detect when a scene is changing in video
I use in command line 

**ffmpeg -i input.flv  -filter:v "select='gt(scene,0.4)',showinfo"  -f null  - 2> ffout**

but I want to do it with node-fluent-ffmpeg

thanks in advance

**Comment by smithaitufe** [comment]
Were you able to resolve this?

**Comment by samuelcalegari** [comment]
no, I didn't succeed with node-fluent-ffmpeg. finally I have used an command execution with node.js :(

**Comment by tomerb78** [comment]
any update on this?
getting error:
"ffmpeg exited with code 1: Error opening filters!"

**Comment by LuciferSharma** [comment]
Any updates on this?
stderrLine gives line by line ffmpeg logs but It misses most of the frame =  lines
